tutorials/hyperspectral/Plot-Hyperspectral-Spectra.Rmd

```diff
@@ -39,7 +39,7 @@ preferably, RStudio loaded on your computer.
 
 
 ### Data to Download
-{% include/dataSubsets/_data_Imaging-Spec-Data-H5.html %}
+{% include/dataSubsets/_data_Imaging-Spec-Data-H5-2020.html %}
 
```

Comment:
Does this need the note about spectral filtering too?

Reply:
I think that this note about spectral subsetting is in the _includes file in one of the other PRs, but I have lost track at this point. I have a note for myself that, once these PRs are merged, I will double check the final _includes and how it relates to each tutorial, and update as needed.